bio2zarr/cli.py

```diff
@@ -149,6 +149,14 @@ def list_commands(self, ctx):
     help="An approximate bound on overall memory usage (e.g. 10G),",
 )
 
+local_alleles = click.option(
+    "--local-alleles",
```

Comment:
Maybe an explicit flag like ``@click.option('--local-alleles/--no-local-alleles', default=True)`` would be better? I think this is what click likes, and the style I've adopted elsewhere in the CLI.

https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags